homeassistant/components/dsmr_reader/definitions.py

```diff
@@ -119,6 +119,11 @@ def tariff_transform(value):
         "icon": "mdi:flash",
         "unit": ELECTRICAL_CURRENT_AMPERE,
     },
+    "dsmr/reading/timestamp": {
```

Comment:
could this use DEVICE_CLASS_TIMESTAMP? whats the return value format?

Reply:
It's like this one:
```dsmr/consumption/gas/read_at 2020-11-08T14:20:02+01:00```

Actual reading:
```dsmr/reading/timestamp 2020-11-08T13:23:57Z```

Reply:
@ctalkington that's actually a good idea! I've added `DEVICE_CLASS_TIMESTAMP` to "dsmr/reading/timestamp" and "dsmr/consumption/gas/read_at"